src/pytorch_lightning/tuner/batch_size_scaling.py

```diff
@@ -234,10 +234,12 @@ def _adjust_batch_size(
     """
     model = trainer.lightning_module
     batch_size = lightning_getattr(model, batch_arg_name)
+    assert batch_size is not None
     new_size = value if value is not None else int(batch_size * factor)
     if desc:
         log.info(f"Batch size {batch_size} {desc}, trying batch size {new_size}")
 
+    assert trainer.train_dataloader is not None
```

Comment:
Hi @ar90n , it seems that tests started failing on this check. Can you please take a look?

Reply:
Hi @otaj. In my understanding, these errors will be fixed in  #11089. So I think that this PR will be merged after #11089 is merged or will be closed without merging.

I'm so sorry if my understanding is wrong. If I should fix these errors. I'm going to try. But if so, this PR may contain a lot of modifications that are not for type hints.

Reply:
Hi @ar90n, you are absolutely correct, sorry about the noise. I should keep myself more up to date with other PRs :D